static/src/components/AdditionalWords/AdditionalWords.ts

```diff
@@ -0,0 +1,145 @@
+import Words from "../api/Words";
+import Control from "../common/control";
+import Input from "../common/Input";
+import Logging from "../login/Logging";
+import ErrorNewWord from "../utils/ErrorNewWord";
+import { FORM_INPUTS_FILE, FORM_INPUTS_TEXT } from "./formInputs";
```

Comment:
Correct path for module in this string

It should be "./FormInputs", not "./formInputs"

Reply:
Fixed. Done